src/index.html

```diff
@@ -19,6 +19,5 @@ <h1 class="logo">To do List</h1>
             <button type="submit" id="btnDelComplete" >Clear all completed</button>
         </div>
         </main>
-        <footer></footer>
```

Comment:
As stated in the previous review, please remove this footer.

Reply:
Kindly I used the Footer and header as Ribbons Asides achieving semantical and accessibility concerns